protocol/x/vault/keeper/shares.go

```diff
@@ -127,7 +175,37 @@ func (k Keeper) MintShares(
 		return err
 	}
 
-	// TODO (TRA-170): Increase owner shares.
+	// Increase owner shares in the vault.
+	ownerShares, exists := k.GetOwnerShares(ctx, vaultId, owner)
+	if !exists {
+		// Set owner shares to be sharesToMint.
+		err := k.SetOwnerShares(
+			ctx,
+			vaultId,
+			owner,
+			types.BigRatToNumShares(sharesToMint),
+		)
+		if err != nil {
+			return err
+		}
+	} else {
+		// Increase existing owner shares by sharesToMint.
+		existingOwnerShares, err := ownerShares.ToBigRat()
+		if err != nil {
+			return err
+		}
+		err = k.SetOwnerShares(
+			ctx,
+			vaultId,
+			owner,
+			types.BigRatToNumShares(
+				existingOwnerShares.Add(existingOwnerShares, sharesToMint),
```

Comment:
Non-blocking: Since we always store integer denominators / numerators, both these integers can grow fairly quickly if an owner gets shares with denominators that are co-prime / has a large lowest common factor with the existing shares.
E.g. User has 1/1001 shares, deposits to get 1/1002 shares, now the denominator is 1001*1002.
Discussed this with @tqin7 offline, think this may not be an issue short-term but definitely something to think about long-term.